openedx/core/djangoapps/user_api/accounts/tests/test_api.py

```diff
@@ -231,8 +231,8 @@ def test_create_account(self):
             'bio': None,
             'profile_image': {
                 'has_image': False,
-                'image_url_full': 'http://example-storage.com/profile_images/default_50.jpg',
-                'image_url_small': 'http://example-storage.com/profile_images/default_10.jpg',
+                'image_url_full': '/static/default_50.png',
+                'image_url_small': '/static/default_10.png',
```

Comment:
Good question. It used to be that the default profile image was assumed to be on the CDN, but now we get its URL from the Django staticfiles loader. In tests, all URLs from that loader are returned as starting with ``/static/``.